apps/aot/src/app/leaderboard/_components/daily-leaderboard.tsx

```diff
@@ -3,6 +3,8 @@ import { DataTableLeaderboard } from '@repo/ui/components/data-table-leaderboard
 import { ADVENT_CHALLENGE_IDS, LEADERBOARD_RANKING_LIMIT } from '~/app/leaderboard/constants';
 import { dailyLeaderboardColumns } from './columns';
 
+export const dynamic = 'force-dynamic';
+
 const getFirst100SubmissionsRanked = async (adventDay: number) => {
   const challengeId = ADVENT_CHALLENGE_IDS[adventDay - 1];
```

Comment:
I dont think we can make this a constant, id's will vary potentially, we'll need to populate this based on a db call. 

for example you can take the year from the url and find the 2024 advent of typescript track and grab all the challenge ids from there

Reply:
Hmm, are you sure? Isn't the list of challenges for AoT constant? Are you saying they're going to change?

For the queries I have below, we need challengeId's, since that's what we have an index on, not slug.

So I was thinking if we know what the challengeId's are for all of AoT, we can just have it like this:

`const ADVENT_CHALLENGE_IDS = [day_1_id, day_2_id, ...]`

and then use this as a reference. Let me know if I'm completely misunderstanding here.